bindings/python/test/trajectory/test_state.py

```
@@ -592,3 +592,39 @@ def test_extract_coordinates(
         )
         assert len(pv_coordinates) == 6
         assert (pv_coordinates == state.get_coordinates()).all()
+
+    def test_is_near(
+        self,
+        state: State,
+    ):
+        tolerance_map: dict[CoordinateSubset, float] = {
```

Ruff (0.8.2)

Undefined name `CoordinateSubset`

(F821)